python/tvm/topi/arm_cpu/dense_alter_op.py

```diff
@@ -52,23 +54,25 @@ def _alter_dense(attrs, inputs, tinfos, out_type):
         ), "matmul_sme.arm_cpu requires weights be a Relay Constant"
 
         weight_dtype = tinfos[1].dtype
-        weight_data = inputs[1].data.numpy()
-        interleaved = weight_data.transpose()
-        encoded_weight = relay.const(interleaved, weight_dtype)
+        encoded_weight = inputs[1]
+        transpose_b = True
```

Comment:
Maybe worth adding a comment there why we need to do these conditional transposes?